src/app/api/v1/admin/users/[primarySha1]/route.ts

```diff
@@ -117,15 +123,34 @@ export async function PUT(
       for (const action of actions) {
         switch (action) {
           case "subscribe": {
+            const sub = await getSubscriberByFxaUid(subscriber.fxa_uid!);
+            const currentFxAProfile = sub.fxa_profile_json as FxaProfile;
+            const subscriptions = new Set(currentFxAProfile.subscriptions);
+
+            subscriptions.add(MONITOR_PREMIUM_CAPABILITY);
+            // update fxa profile data to match subscription status
+            currentFxAProfile.subscriptions = Array.from(subscriptions);
+            await updateFxAProfileData(sub, JSON.stringify(currentFxAProfile));
+
             // activate and opt out profiles
-            await activateAndOptoutProfile(onerepProfileId);
+            await activateProfile(onerepProfileId);
+            await optoutProfile(onerepProfileId);
```

Comment:
Any reason to split this up again? Does that reason also hold for other invocations of `activateAndOptoutProfile`? (i.e. should that function be removed?)

Reply:
I originally used `activateAndOptoutProfile` but it does an early return here:

https://github.com/mozilla/blurts-server/blob/8e6a071250230e2a2a7e90dd7245b86c0e64edc8/src/app/functions/server/onerep.ts#L246-L248

We should rethink how this works and is used by callers, I agree it's confusing having them be separate methods.

Reply:
I can file an issue to refactor, I think this might bite someone in the future.